travel-agency-spring-mvc/src/main/java/cz/fi/muni/pa165/travelagency/mvc/controller/CustomerController.java

```diff
@@ -37,6 +40,13 @@
 
     final static Logger log = LoggerFactory.getLogger(CustomerController.class);
 
+    @InitBinder
+	protected void initBinder(WebDataBinder binder) {
+		if (binder.getTarget() instanceof CustomerDTO) {
+            binder.addValidators(new CustomerDTOValidator());
+        }
```

Comment:
Týmto inicializuješ custom Spring validáciu a kedže ju netreba tak to môžeš tiež vymazať